omod/src/test/java/org/openmrs/module/appointments/web/controller/RecurringAppointmentsControllerIT.java

```diff
@@ -484,6 +486,9 @@ public void shouldReturnNoContentResponseForRecurringAppointmentsForNoConflicts(
     }
 
     @Test
+    @Ignore
```

Comment:
@N0-man can we have a tech debt card created in the backlog to handle this later point of time ?

Reply:
yes [BAH-2300](https://bahmni.atlassian.net/browse/BAH-2300)